File: Proyecto-DABD-ADFS-main/HotelesAPI/Models/Usuario.cs
namespace HotelesAPI.Models
{
    public class Usuario
    {
        public int IdUsuario { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string? PasswordHash { get; set; }
        public string Telefono { get; set; } = string.Empty;
        public string Rol { get; set; } = "cliente";
        public bool Activo { get; set; } = true;
        public DateTime FechaRegistro { get; set; }
    }
}

File: sat-backend/src/main/java/com/halcon/sat/model/Usuario.java
package com.halcon.sat.model;

import java.time.LocalDateTime;

public class Usuario {

    private Long idUsuario;
    private String email;
    private String passwordHash;
    private String nombres;
    private String apellidos;
    private String tipoUsuario;
    private boolean activo;
    private LocalDateTime fechaRegistro;

    public Long getIdUsuario() { return idUsuario; }
    public void setIdUsuario(Long idUsuario) { this.idUsuario = idUsuario; }

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }

    public String getPasswordHash() { return passwordHash; }
    public void setPasswordHash(String passwordHash) { this.passwordHash = passwordHash; }

    public String getNombres() { return nombres; }
    public void setNombres(String nombres) { this.nombres = nombres; }

    public String getApellidos() { return apellidos; }
    public void setApellidos(String apellidos) { this.apellidos = apellidos; }

    public String getTipoUsuario() { return tipoUsuario; }
    public void setTipoUsuario(String tipoUsuario) { this.tipoUsuario = tipoUsuario; }

    public boolean isActivo() { return activo; }
    public void setActivo(boolean activo) { this.activo = activo; }

    public LocalDateTime getFechaRegistro() { return fechaRegistro; }
    public void setFechaRegistro(LocalDateTime fechaRegistro) { this.fechaRegistro = fechaRegistro; }
}
